android/src/main/java/com/urbanairship/reactnative/UrbanAirshipReactModule.java

```diff
@@ -556,6 +556,7 @@ public void setQuietTime(ReadableMap map) {
         start.set(Calendar.MINUTE, startMinute);
         end.set(Calendar.HOUR_OF_DAY, endHour);
         end.set(Calendar.MINUTE, endMinute);
+        //noinspection deprecation
         UAirship.shared().getPushManager().setQuietTimeInterval(start.getTime(), end.getTime());
```

Comment:
Deprecated api still being used and exposed. I deprecated the js side so we can remove this eventually.